shared/js/background/storage/tds.es6.js

```diff
@@ -12,12 +12,29 @@ class TDSStorage {
         this.dbc.version(1).stores({
             tdsStorage: 'name,data'
         })
+
         this.tds = { entities: {}, trackers: {}, domains: {}, cnames: {} }
         this.surrogates = ''
         this.ClickToLoadConfig = {}
         this.config = { features: {} }
+
         this.isInstalling = false
-        this.onUpdatedListeners = new Map()
+
+        this._onUpdatedListeners = new Map()
+
+        this._onReadyResolvers = new Map()
+        this._onReadyPromises = new Map()
+        for (const configName of
+            ['tds', 'surrogates', 'ClickToLoadConfig', 'config']) {
```

Comment:
I think this can be derived from tdsLists[].name from constants.

Reply:
Good point! Done